.travis.yaml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+env:
+  - CI=1
+language: ruby
+bundler_args: --without debug
+rvm:
```

Comment:
The reason I wouldn't support is this:

https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/
Support for Ruby version 1.9.3 will end on February 23, 2015